content/arduino-cloud/03.tutorials/document-projects-through-webeditor/classroom_project.md

```diff
@@ -0,0 +1,103 @@
+
+![proximity_meter_real_components.png](assets/classroom_project_photo_proximity_meter.png)
+
+### 3. Open a new sketch.
+Open the Web Editor by clicking on the ![header_dropdown_icon.png](assets/icon_header_dropdown.png) icon in the header. 
+
+![proximity_meter_real_components.png](assets/classroom_project_webeditor.png)
+
+When your editor is ready, create a new sketch by clicking ![new_sketch_icon.png](assets/icon_new_sketch.png) and rename the sketch to the name of your project. 
+
+![classroom_project_rename_sketch](assets/classroom_project_rename_sketch.png)
+
+### 4. Write your sketch. 
+Write your sketch or copy-paste your sketch from another project. For easy readibility ensure that
+* Adding comments to describe important statements 
+* Assigning good varible names 
+* Inundating the code 
+
+Here's the example sketch of the Proximity Meter project for your reference.  
+
+<div class="sketch-iframe" data-src="https://create.arduino.cc/editor/ArduinoEdu/e833b75d-5989-4049-aec4-a2472303415d/preview?embed&snippet=L73-L77&hideNumbers&height=120#L73-L77"></div>
+
+## Documenting your Project 
+To use the project, students must be able to clearly understand the goals of the project and they must do in each step of making the project. The web editor comes with a ReadMe.adoc file attached to the sketch. Here you can document all the intructions required. The template comes with these sections 
+* Credits
+* Project description
+* Instructions 
+* Troubleshooting tips
+
+### 5. Fill in your credits.
+The first part of the Readme are credits where you can fill in your personal details. Students can use this section to reach out to you if needed. 
+
+´´´ arduino
+:Author: LenGeo
+:Email: {AuthorEmail}
+:Date: 22/02/2023
+:Revision: version#
+:License: Public Domain
+´´´
+![classroom_project_documentation_credentials](assets/classroom_project_documentation_credentials.png)
+
+[Note] - license is something you don’t have to be concerned with unless you want to share your project on the internet 
+
+### 6. Describe your project
+Give a short description of the project. In this example we will describe what the Proximity Meter is going to do. 
+
+![classroom_project_documentation_description](assets/classroom_project_documentation_description.png)
+
+### 7. Provide instructions 
+Briefly describe what you intend your students to do with this project. Make sure you provide all details right from intalling the right plugins to uploading the code to the board. 
+
+![classroom_project_documentation_instructions](assets/classroom_project_documentation_instructions.png)
+
+Note- Most students would be new to the web editor and they wouldn’t have the webeditor installed. Copy paste this link if would like to https://docs.arduino.cc/arduino-cloud/getting-started/getting-started-web-editor
```

Comment:
You can't install the Web Editor - what is it you are trying to say here? Also I think the sentence stops too early?

Reply:
Ah I meant to install the plugin.